BackendCommands.php

```diff
@@ -134,43 +118,162 @@ public function updateCode()
      */
     public function updateDatabase()
     {
-        $this->drush($this->selfRecord(), 'updatedb', [], ['yes' => true]);
+        $this->drush($this->selfRecord(), 'updatedb', [], ['yes' => $this->input()->getOption('yes')]);
         $this->drush($this->selfRecord(), 'cache:rebuild');
-        $this->drush($this->selfRecord(), 'locale-update', [], ['yes' => true]);
+        $this->drush($this->selfRecord(), 'locale-update', [], ['yes' => $this->input()->getOption('yes')]);
+    }
+
+    /**
+     * Runs populateConfigSyncDirectory() for backend:config-export.
+     *
+     * As long as we have to handle environment specific config, this can not
+     * be a post command for the default drush config:import command.
+     *
+     * TODO: Revisit when we do not need the local- and testing-environment config
+     * TODO: folder anymore. Then decide if we can make this a post command for config:*.
+     *
+     * @hook pre-command backend:config-export
+     *
+     * @param \Consolidation\AnnotatedCommand\CommandData $commandData
+     */
+    public function preConfigExportCommand(CommandData $commandData)
+    {
+        $this->populateConfigSyncDirectory();
     }
 
     /**
      * Export configuration of BurdaStyle backend site.
      *
+     * TODO: Revisit when we do not need the local- and testing-environment config
+     * TODO: folder anymore. Then decide if we can make this a post command for config:export.
+     *
      * @command backend:config-export
      *
      * @aliases backend:cex
      *
      * @options-backend
      *
-     * @validate-site-alias
-     *
-     * @usage drush @elle backend:export-config
+     * @usage drush @elle backend:config-export
      *   Exports the elle configuration.
      *
      * @bootstrap config
      */
     public function configExport()
     {
+        if ($this->environment !== 'prod') {
+            $this->logger()->error('Only production will be exported. Current environment is "%environment".', ['%environment' => $this->environment]);
```

Comment:
What is that doing?

Reply:
It is preventing, that config from local is exported